code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -401,13 +401,15 @@
 	color = "#000000"
 	taste_description = "dust"
 	goal_difficulty = REAGENT_GOAL_EASY
+	unable_to_purge = TRUE
```

Comment:
perhaps rename to `purgeable` for readability.



Example:

`R.purgeable` vs. `R.unable_to_purge`